ark/utils/segmentation_utils.py

```diff
@@ -180,3 +182,47 @@ def concatenate_csv(base_dir, csv_files, column_name="fov", column_values=None):
             combined_data = pd.concat((combined_data, temp_data), axis=0, ignore_index=True)
 
     combined_data.to_csv(os.path.join(base_dir, "combined_data.csv"), index=False)
+
+
+def visualize_segmentation(segmentation_labels_xr, channel_data_xr,
+                           output_dir, chan_list=None, fovs=None, show=False):
+    """For each fov, generates segmentation labels, segmentation borders, and overlays
+    over the channels in chan_list if chan_list is provided.
+
+    Saves overlay images to output directory
+
+    Args:
+        segmentation_labels_xr (xarray.DataArray):
+            xarray containing segmentation labels
+        channel_data_xr (xarray.DataArray):
+            xarray containing TIFs
+        output_dir (str):
+            path to directory where the output will be saved
+        chan_list (list):
+            list of channels to overlay segmentation output over
+        fovs (numpy.ndarray):
+            field of view
```

Comment:
The description for FOVs isn't very helpful for figuring out what the purpose of this argument is